custom_components/powercalc/data/linkind/ZBT-DIMLight-D0120/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plus Plug S",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 1,
+        "VERSION": "v1.9.2:docker"
+    },
+    "name": "Dimmable light (ZBT-DIMLight-D0120)",
```

As found here: https://specified.tech/en-us/product/970/dimmablewarmwhitefilamente2762w806lmbulb

The name will be shown in supported models listing.
And model will be picked from the directory name.
So we provide as much information about the light to the user this way. 

```suggestion
    "name": "Dimmable Warm White Filament E27 6.2W 806lm Bulb",
```